spec/features/admin/orders_spec.rb

```diff
@@ -169,6 +169,9 @@ def new_order_with_distribution(distributor, order_cycle)
 
     expect(page).to have_selector 'h1.page-title', text: "Customer Details"
 
+    # The customer selection partial should be visible
+    expect(page).to have_selector '#select-customer'
```

Comment:
At this point in the test in `master`, the order state is `cart`, but in the 3-0 branch it's `address`. This PR fixes the spec, but this indicates that order states are not changing in the same way they did before. I'm not sure if there are other implications to that...